xmake/toolchains/llvm/xmake.lua

```diff
@@ -30,9 +30,9 @@ toolchain("llvm")
 
     -- set toolset
     set_toolset("cc",     "clang")
-    set_toolset("cxx",    "clang", "clang++")
-    set_toolset("mxx",    "clang", "clang++")
-    set_toolset("mm",    "clang", "clang++")
+    set_toolset("cxx",    "clang++", "clang")
```

Comment:
why? clang can also compile c++ code.

Reply:
well it's more logical to use clang++ when compiling c++ :D

Reply:
I don't think it should be modified unless it's to fix some unavoidable bug.

Tools like autotools, boost/b2, etc. can detect gcc better, and I'm not sure how much passing in g++/clang++ would break anything. It would introduce a lot of uncertainty.